### ISSUE TITLE
Modifying task properties by submitting a comment

### ISSUE THREAD

**Issue by nioncode** [kanboard/kanboard#4018]
It would be great if there would be some kind of action syntax that allows changing properties of a task when submitting a comment.
Today went through a few of my tasks and wrote things like 'fixed in rev xyz', 'won't fix, because of abc', 'duplicate of #uvw', and for each of those I had to first write the comments and then modify the task myself.

Examples of desired syntax (commands are only parsed on new lines):

Closing a task:
```
/close
```

Changing assignee:
```
/assign user
```

Changing due date:
```
/due x days
/due 24-10
```

Adding tags:
```
/tag one two three
```

Changing Priority:
```
/priority level
```

Changing Category:
```
/category name
```

Duplicates:
```
/duplicate #xyz
```

There are probably many more examples, so it would be great if we could gather a list and decide if this is something that should be implemented.

**Comment by creecros** [comment]
Sounds cool, you developing it?
https://docs.kanboard.org/en/latest/plugins/index.html

**Comment by nioncode** [comment]
No, but I might take a shot at it when I have some spare time if nobody else starts to work on this.

@everyone: if you start to implement this, please ping me so that we don't implement this twice

**Comment by nioncode** [comment]
Today, I implemented a rough sketch for a plugin to handle this.

Remaining questions:
1. Which actions should be supported?
1. @fguillot Is it possible to provide a list of possible actions (like for user mentions) whenever you type a '/' on a new line from a plugin or must this be implemented in core?

I will publish the plugin as a preview after I cleaned it up a bit and implemented a few more actions.
